src/web/app/components/question-responses/gqr-rqg-view-responses/gqr-rqg-view-responses.component.html

```diff
@@ -76,7 +80,11 @@ <h3>{{ teamInfo.key }} Detailed Responses</h3>
           <div class="card-header bg-info text-white" (click)="question.isTabExpanded = !question.isTabExpanded">
             <tm-question-text-with-info [questionNumber]="question.questionOutput.feedbackQuestion.questionNumber"
                                         [questionDetails]="question.questionOutput.feedbackQuestion.questionDetails"
-                                        (click)="$event.stopPropagation()"></tm-question-text-with-info>
+                                        (click)="$event.stopPropagation()"
+                                        (downloadQuestionResultEvent)="triggerDownloadQuestionResult(
+                                        question.questionOutput.feedbackQuestion.questionNumber,
+                                        question.questionOutput.feedbackQuestion.feedbackQuestionId);"
```

Comment:
Similar here

Reply:
The indentations follow that of the components below/above it. Maybe it appears weirdly in the git viewer?